library/src/main/java/com/schibsted/spain/barista/BaristaListViewActions.kt

```diff
@@ -0,0 +1,94 @@
+package com.schibsted.spain.barista
+
+import android.support.annotation.IdRes
+import android.support.test.espresso.Espresso.onData
+import android.support.test.espresso.NoMatchingViewException
+import android.support.test.espresso.matcher.ViewMatchers.*
+import android.view.View
+import android.widget.AdapterView
+import com.schibsted.spain.barista.custom.PerformClickAction.clickUsingPerformClick
+import com.schibsted.spain.barista.internal.failurehandler.SpyFailureHandler
+import com.schibsted.spain.barista.internal.failurehandler.withFailureHandler
+import org.hamcrest.Matchers.*
+
+object BaristaListViewActions {
+
+    @JvmStatic
+    fun clickListViewItem(@IdRes listViewId: Int, vararg positions: Int) {
```

Comment:
Not related with this PR... but I think that this method is useless. I wrote it because I thought it was an smart idea... but I can't think in a usecase for it. Just code that has to be understood and maintained.